app/renderer/components/textbox.js

```diff
@@ -0,0 +1,87 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const React = require('react')
+const ImmutableComponent = require('../../../js/components/immutableComponent')
+const {StyleSheet, css} = require('aphrodite')
+const globalStyles = require('./styles/global')
+
+class Textbox extends ImmutableComponent {
+  render () {
+    const className = css(
+      this.props['data-isFormControl'] && styles.formControl,
+      styles.textbox,
+      this.props['data-isSettings'] && styles.isSettings,
+      (this.props.readonly || this.props.readOnly) ? styles.readOnly : styles.outlineable,
+      this.props['data-isRecoveryKeyTextbox'] && styles.recoveryKeys
+    )
+
+    return <input type='text' className={className} {...this.props} />
+  }
+}
+
+class FormTextbox extends ImmutableComponent {
+  render () {
+    return <Textbox data-isFormControl='true' {...this.props} />
+  }
+}
+
+class SettingTextbox extends ImmutableComponent {
+  render () {
+    return <FormTextbox data-isSettings='true' {...this.props} />
+  }
+}
+
+class RecoveryKeyTextbox extends ImmutableComponent {
+  render () {
+    return <SettingTextbox data-isRecoveryKey='true' {...this.props} />
+  }
+}
+
+const styles = StyleSheet.create({
+  'formControl': {
```

Comment:
Can we share this with [that](https://github.com/brave/browser-laptop/pull/6689/files#diff-fc51489835db20322f43545a85d88849R40) of app/renderer/components/dropdown.js? I'm wondering if setting them in each file would make a possibility of style inconsistency.